### PR TITLE
ML 2.2 : Contrast between Regression and Classification (V)

### LINKED ISSUE #2786: ML 2.2 : Contrast between Regression and Classification (V)
### Description

Welcome to 'ML' Team, good to see you here

This issue will helps readers in acquiring all the knowledge that one needs to know about **Contrast between Regression and Classification**.

To get assigned to this issue, add your **Batch Numbers** mentioned in the spreadsheet of "Machine Learning", this issue is only for **Video** content.

Domain : Machine Learning

Points to Note :

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.

- "Issue Title" and "PR Title should be the same. Include issue number along with it.

- Changes should be made inside the `Machine Learning/Machine_Learning/Supervised_Machine_Learning` Branch.

- Follow Contributing Guidelines & Code of Conduct before start Contributing.

- This issue is only for 'GWOC' contributors of 'Machine Learning' domain.

**To be Mentioned while taking the issue :**
- Full name
- Batch Number
- GitHub Profile Link
- Which type of Contribution you want to make : 
    - [ ] Video

All the best. Enjoy your open source journey ahead. 😎

### Domain

Machine Learning

### Type of Contribution

Video

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Sumit3301**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #2786 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [x] Machine Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸
image

 
<hr>

### CODE REVIEW

**Comment by Sumit3301** [comment]
@geeythree  is this correct? kindly check

**Comment by Sumit3301** [comment]
@Isha307  can you please check and let me know if this is correct?

File: Machine_Learning/Supervised_Machine_Learning/ML_2_2_Contrast between Regression and Classification (V).md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+### Contrast between Regression and Classification
+Things I have discussed:
+1. What is regression
+2. Types of regression
+3. Where it is used.
+4. Implementation of liner regression.
```
Comment (Isha307) [inline Machine_Learning/Supervised_Machine_Learning/ML_2_2_Contrast between Regression and Classification (V).md:6]: linear

File: Machine_Learning/Supervised_Machine_Learning/ML_2_2_Contrast between Regression and Classification (V).md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+5. Mathematics behind regression
+6. What is classification
+7. Where it is used
+8. Basic mathematics behind logistic regression
+9. Difference between both
```
Comment (Isha307) [inline Machine_Learning/Supervised_Machine_Learning/ML_2_2_Contrast between Regression and Classification (V).md:11]: replace both with "classification and regression"

**Review by Isha307 — CHANGES_REQUESTED** [review]

**Comment by Isha307** [comment]
@Sumit3301 also change the PR title to this "ML 2.2 : Contrast between Regression and Classification (V)"

**Comment by Sumit3301** [comment]
> @Sumit3301 also change the PR title to this "ML 2.2 : Contrast between Regression and Classification (V)"

done

**Review by Isha307 — APPROVED** [review]

**Comment by Isha307** [comment]
@Sumit3301 Congratulation🥳, your first PR got merged.

**Pull request merged**